configure.ac

```diff
@@ -86,6 +86,31 @@ AS_IF([test "x$android" = "xno"], [
   AC_DEFINE(HAVE_PROCMEM, [0], [Enable /proc/pid/mem support])
 ])
 
+# Check for termcap and readline or bypass checking for the libraries.
+AC_ARG_WITH([readline], [AS_HELP_STRING([--without-readline],
+                            [build without readline])])
+AM_CONDITIONAL([WITH_READLINE], [test "x$with_readline" != "xno"])
+AS_IF([test "x$with_readline" != "xno"], [
+  # termcap is sometimes required by readline
+  AC_CHECK_LIB([termcap], [tgetent], [], [])
+  AC_CHECK_LIB([readline], [readline], [], [
+    echo "libreadline could not be found, which is required to continue."
+    echo "The libreadline-dev package may be required."
+    exit 1
+  ])
+])
+
+GETTEXT_PACKAGE=GameConqueror
```

Comment:
Uhm, sorry it took a while, I indeed had leftovers from previous configures and removed manually some files, not a good approach.

I can get the gui-less build to go on if I also remove intltoolize from autogen.sh, but this will get complicated for no gain.

Let's remember that we call intltool even if not needed (either in a comment or the commit message) and let's call it a day.